docs/getting-started/gs-credentials.md

```diff
@@ -48,7 +48,9 @@ If you're using a client-side implementation (see [UID2 Client-Side Integration
 - **Subscription ID**: This value identifies your site to the UID2 service.
 - **Public key**: This value is used for encryption.
 
-When you're implementing UID2 on the client side, by using the UID2 JavaScript SDK or Prebid, provide the values to the SDK or to Prebid as part of configuration.
+**Client keypair** is a group term we use for these two values, which are used to uniquely define an account that's using an implementation that generates the token on the client side.
+
+When you're implementing UID2 on the client side, by using the UID2 JavaScript SDK, Client-Side Integration for Mobile, or Prebid, provide the values to the SDK or to Prebid as part of configuration.
```

Comment:
i think we can say Prebid.js instead of Prebid here - for using Prebid Mobile, they use keyapir when following Client-Side Integration for Mobile

Reply:
Thx. Fixed two instances on this line.